src/player.ts

```diff
@@ -437,6 +437,20 @@ export class Player extends NativeInstance<PlayerConfig> {
     return PlayerModule.getAvailableVideoQualities(this.nativeId);
   };
 
+  /**
+   * Sets the video quality.
+   * ONLY works on Android. No effect on iOS and tvOS devices.
+   */
```

Comment:
Please also document the parameter. The most important part here is to mention the special value `'auto'`!

```suggestion
   *
   * @param qualityId value obtained from `VideoQuality.id`, which can be obtained via `Player.getAvailableVideoQualities()` to select a specific quality. To use automatic quality selection, 'auto' can be passed here.
   */
```